lib/cart.rb

```diff
@@ -0,0 +1,27 @@
+require_relative "./item"
+
+class Cart 
+
+    attr_reader :items , :items_bought_list
+
+    def initialize
```

Comment:
Initialize only the class instance variables like items and items_bought_list. Any other action needs to be done in separate methods, that need to be called whenever appropriate.